tests/kafkatest/services/performance/producer_performance.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, context, num_nodes, kafka, security_protocol, topic, num_reco
 
     def _worker(self, idx, node):
         args = self.args.copy()
+<<<<<<< HEAD
```

Comment:
oops..sorry for this clear error. I have tested the ProducerPerformance code. As i am not ready with my setup, I have not tested system test.